GUI/src/main/java/main/Drifty_GUI.java

```diff
@@ -50,12 +50,17 @@ public void init() {
 
     @Override
     public void start(Stage primaryStage) {
-        this.primaryStage = Constants.getStage("Drifty GUI", true);
-        this.primaryStage.setMinWidth(Constants.SCREEN_WIDTH * .46);
-        this.primaryStage.setMinHeight(Constants.SCREEN_HEIGHT * .8125);
-        createScene();
-        this.primaryStage.setScene(scene);
-        this.primaryStage.show();
+
+        try {
+            this.primaryStage = Constants.getStage("Drifty GUI", true);
+            this.primaryStage.setMinWidth(Constants.SCREEN_WIDTH * .46);
+            this.primaryStage.setMinHeight(Constants.SCREEN_HEIGHT * .8125);
+            createScene();
+            this.primaryStage.setScene(scene);
+            this.primaryStage.show();
+        } catch (Exception e) {
+            msgBroker.msgLogError("Error starting Drifty GUI Application");
```

Comment:
The current error message is too vague. It only indicates that an exception occurred, without revealing the actual cause. Including the exception message (`e.getMessage()`) would make it much easier to understand the real issue.